test/PackageManagement.Test/BuildIntegratedRestoreUtilityTests.cs

```diff
@@ -408,6 +356,81 @@ public class BuildIntegratedRestoreUtilityTests
         }
 
         [Fact]
+        public async Task CacheHasChanges_ReturnsTrue_IfSupportProfilesDiffer()
```

Comment:
For naming consistency in the file, could you rename to BuildIntegratedRestoreUtility_CacheDiffersOnSupportProfiles?